deployment/data-streams/changeset/sequence/deploy_data_streams.go

```diff
@@ -0,0 +1,312 @@
+package sequence
```

Comment:
As we will have a sequence coming from the operations API, this could be confusing.

Reply:
Could be - I think we can move it out if it becomes confusing